CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+## [v0.62.0-rc.0.0]
+
+### Breaking
+
+- Removing <TextInput>'s `onTextInput` event ([3f7e0a2](https://github.com/facebook/react-native/commit/3f7e0a2) by [@shergin](https://github.com/shergin))
```

Comment:
Change the verb case; prefer the imperative (implied "You") rather than present continuous (implied "I am...").

```suggestion
- Remove <TextInput>'s `onTextInput` event ([3f7e0a2](https://github.com/facebook/react-native/commit/3f7e0a2) by [@shergin](https://github.com/shergin))
```